hannakim91/romcom
# Work on make your own cover button

Can you review the changes Amy?

### What's this PR do? 
1. Added functionality to "Make your own cover" button.
2. Created query selectors relevant to function.
3. Created a function that responds by showing hidden form section and hiding home section.
4. Created an event listener to respond to user clicking on "make your own cover" button with above function.

### Where should the reviewer start?
Lines:
7-12: query selectors
22: event listener
39-42: function

### How should this be manually tested?
Check functionality of "Make your own cover" button - click button and see if form section shows.
